data/debian/sonic-host-services-data.determine-reboot-cause.service

```diff
@@ -1,12 +1,13 @@
 [Unit]
 Description=Reboot cause determination service
-Requires=rc-local.service database.service
-After=rc-local.service database.service
+Requires=rc-local.service
+After=rc-local.service
+ConditionPathExists=!/tmp/determine-reboot-cause
```

Comment:
The process won't restart but just in case someone restarts it manually or my error the script code creates an unwanted reboot entry as the script does not handle the restart case. I think keeping this conditional check is safe and prevents any false restart. No harm in keeping it.

Reply:
@anamehra no that is not a valid scenario to be taken care. someone running the service manually is outside the scope. The unit file is for orchestration by systemd